packages/scaffolding-cli/templates/test/lighthouse-ci/ReadMe.md

```diff
@@ -0,0 +1,71 @@
+
+1. Pull the container down and run by passing through the lhci config:
+  `docker run -v "$(pwd)/lighthouserc.json:/opt/lhci/lighthouserc.json" -v "$(pwd)/results:/opt/lhci/.lighthouseci" --rm -i amidostacks/lhci:latest lhci collect --config=lighthouserc.json`
+
+### Local
+
+1. Build the image or, pull the latest from [amidostacks/node](https://hub.docker.com/r/amidostacks/node).
+  `docker image build -t amidostacks/lhci .`
+
+2. Run the container named lhci and pass through the lhci config:
+  `docker run --name lhci -v "$(pwd)/lighthouserc.json:/opt/lhci/lighthouserc.json" -v "$(pwd)/results:/opt/lhci/.lighthouseci" --rm -i -t amidostacks/lhci:latest lhci collect --config=lighthouserc.json`
+
+3. Run the container named lhci and pass through a URL:
+  `docker run --name lhci -v "$(pwd)/lighthouserc.json:/opt/lhci/lighthouserc.json" -v "$(pwd)/results:/opt/lhci/.lighthouseci" --rm -i -t amidostacks/lhci:latest lhci collect --url=https://google.com`
+
+4. This will output the collect results in both html and json foramt to [/results](./results), and the assertions to `.lighthouseci`
+  
+5. To assert on the results against specified baseline of acceptable standards:
+  `docker run --name lhci -v "$(pwd)/lighthouserc.json:/opt/lhci/lighthouserc.json" -v "$(pwd)/results:/opt/lhci/.lighthouseci" --rm -i -t amidostacks/lhci:latest lhci assert --config=lighthouserc.json`
+
+## Lighthouse CI
+
+We are using the [Lighthouse CI](https://github.com/GoogleChrome/lighthouse-ci#readme) for collecting Lighthouse reports. For more information on Lighthouse, see `https://developers.google.com/web/tools/lighthouse/`.
+
+The config related to Google Lighthouse can be found in [./lighthouserc.json](./lighthouserc.json). This includes a template for:
+
+- `lhci collect` - the [collect](https://github.com/GoogleChrome/lighthouse-ci/blob/master/docs/configuration.md#collect) JSON object defines how the test will be executed and against which URL. These can be overriden by using the CLI directly, e.g. `lhci collect --url=https://google.com`
+- `lhci assert` - the [assert](https://github.com/GoogleChrome/lighthouse-ci/blob/master/docs/configuration.md#assertions) JSON object defines the assertions the `lh:assert` step will run.
+
+The configuration looks like this example:
+
+```json
+{
+  "ci": {
+    "collect": {
+      "chromePath": false,
+      "numberOfRuns": 3,
+      "url": "http://sandbox.euw.deploy.systems.next/footer",
+      "settings": {
+        "extraHeaders": "{\"x-next-realm\": \"Next\",  \"x-next-territory\": \"GB\", \"x-next-language\": \"en\"}"
+      }
+    },
+    "assert": {
+      "assertions": {
+        "categories:performance": ["error", { "minScore": 0.85 }]
+          }
+      }
+  }
+}
+```
+
+### More on Lighthouse
+
+Useful [documentation, examples, and recipes](https://github.com/GoogleChrome/lighthouse#docs--recipes) to get you started.
+
+This includes reference on how to test a site with authentication.
```

Comment:
This is really good documentation. Shouldn't we link it to our Docs too?

Reply:
Good shout, will do.